### ISSUE TITLE
Ordering of columns in docs example doesn't appear to be working

### ISSUE THREAD

**Issue by philipmountifield** [bootstrap-vue/bootstrap-vue#3872]
### Describe the bug

When browsing the documentation I came across an example which doesn't appear to be showing as expected.

### Steps to reproduce the bug

1. Go to https://bootstrap-vue.js.org/docs/components/layout#ordering-columns
2. Note that columns do not appear in the expected order

![image](https://user-images.githubusercontent.com/255614/62948676-c067e580-bddc-11e9-935b-143f07c32f08.png)

### Expected behaviour

I expected to see columns **Third, but first** then **First, but unordered** then **Second, but last**

**Environment:**

- Device: Laptop
- OS: Windows 8
- Browser: Chrome
- Version: 76

### Demo link

https://bootstrap-vue.js.org/docs/components/layout/#ordering-columns

**Comment by tmorehouse** [comment]
Duplicate.

Columns without an explicit order will always appear first in flexbox.

**Comment by tmorehouse** [comment]
It might have been an issue brought up on Discord chat, or in another issue.

But it is not an issue... that is just how flexbox works.  If an explicit order is not provided, those elements appear before any elements with an order set.

**Comment by philipmountifield** [comment]
I see that the wording from the example comes from upstream bootstrap. It seems to be a very poor choice of words only leading to confusion.

Thanks for the clarification.

**Comment by philipmountifield** [comment]
For posterity, and the benefit of anyone who comes across this via Google, a good description of why this happens can be found here: https://gedd.ski/post/flexbox-order/

TLDR: the default order for flexbox items is effectively 0.
